index.js

```diff
@@ -29,14 +27,14 @@ function main(fn, fp, data, opts) {
 		});
 	}
 
-	var json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
+	const json = JSON.stringify(data, opts.replacer, opts.indent) + '\n';
 
 	return fn(fp, json, {mode: opts.mode});
 }
 
 module.exports = function (fp, data, opts) {
```

Comment:
Hmm, one thing about fat arrow functions is that they cannot have a name and thus will always show up as an anonymous function. What do you think about naming the functions instead (`module.exports = function writeJsonFile`) so that stack traces looks good?